src/pymedphys/dicom/_level2/header_tweaks.py

```diff
@@ -31,28 +31,29 @@
 
 from ...libutils import get_imports
 
-from .._level1.dicom_create import dicom_dataset_from_dict
+from .._level1.create import dicom_dataset_from_dict
 
 IMPORTS = get_imports(globals())
 
 
-def adjust_machine_name(ds, new_machine_name):
+def adjust_machine_name(dicom_dataset, new_machine_name):
     """Change the machine name within the DICOM header
     """
 
-    new_ds = deepcopy(ds)
+    new_dicom_dataset = deepcopy(dicom_dataset)
```

Comment:
I'm trialling being more verbose `ds` --> `dicom_dataset`

Reply:
All good.

Is `dataset` not sufficient? We are working in the `dicom` pymedphys space. Avoiding redundant labelling is a specific recommendation in the [Python Best of the Best Practices Github guide](https://gist.github.com/sloria/7001839)...

But honestly, I don't really care at all. It would be a stretch to say `dataset` vs. `dicom_dataset` is even a very slight preference of mine.

Reply:
Somewhat relevant: presumably we'll be using `dataset` (rather than `dicom_dataset`) in our custom Dicom classes?

Reply:
> Somewhat relevant: presumably we'll be using dataset (rather than dicom_dataset) in our custom Dicom classes?

yup